resources/assets/less/bem/profile-stats.less

```diff
@@ -39,6 +39,33 @@
     margin-right: 30px;
   }
 
+  &__playtime-box {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: column;
+
+    margin-bottom: 10px;
+  }
+
+  &__playtime {
+    color: @yellow;
+    font-size: 15px;
+    letter-spacing: 2px;
+    font-weight: 200;
+
+    &--number {
+      font-weight: 600;
+    }
+
+    &--small {
+      font-size: 12px;
+    }
+
+    &:nth-child(even) {
+      padding-right: 4px;
```

Comment:
add to `--unit`. And use `content: ' '` at `::after` so it uses correct spacing.